verticapy/connect.py

```diff
@@ -142,25 +130,89 @@ def change_auto_connection(name: str):
         f.close()
     else:
         raise NameError(
-            "The input name is incorrect. The connection '{}' has never been created.\nUse the new_auto_connection function to create a new connection.".format(
+            "The input name is incorrect. The connection '{}' has never been created.\nUse the new_connection function to create a new connection.".format(
                 name
             )
         )
 
+# ---#
+def close_connection():
+    """
+---------------------------------------------------------------------------
+Close the Database connection.
+    """
+    if verticapy.options["connection"]["conn"] and not(verticapy.options["connection"]["conn"].closed()):
+        verticapy.options["connection"]["conn"].close()
+
+# ---#
+def connect(section: str, dsn: str = ""):
+    """
+---------------------------------------------------------------------------
+Connect to the Database.
+
+Parameters
+----------
+section: str / connector
+    Name of the section in the configuration file.
+dsn: str, optional
+    Path to the file containing the credentials. If empty, the 
+    VERTICAPY_CONNECTIONS environment variable will be used.
+    """
+    prev_conn = verticapy.options["connection"]["conn"]
+    verticapy.options["connection"]["conn"] = vertica_conn(section, dsn)
+    verticapy.options["connection"]["dsn"] = dsn
+    verticapy.options["connection"]["section"] = section
+    if prev_conn and not(prev_conn.closed()):
```

Comment:
I think it would be nicer if you closed the previous connection before creating the new one

Reply:
Done; The change will be made in the next commit.